man/man8/zpool-status.8

```diff
@@ -37,12 +37,17 @@
 .Sh SYNOPSIS
 .Nm zpool
 .Cm status
+.Op Fl c Ar script1 Ns Oo , Ns Ar script2 Ns ,… Oc
 .Op Fl dDegiLpPstvx
-.Op Fl T Sy u Ns | Ns Sy d
-.Op Fl c Op Ar SCRIPT1 Ns Oo , Ns Ar SCRIPT2 Oc Ns …
-.Oo Ar pool Oc Ns …
+.Oo Fl j
+.Oo Ns Fl Fl json-int Oc
```

Comment:
If I'm reading the docs correctly, this looks like the preferred way to handle long options.  Optionally we can escape the hyphen, but since we haven't done that elsewhere let's not to stay consistent.  

```suggestion
.Oo Ns Fl -json-int Oc
```

https://mandoc.bsd.lv/mdoc/style/options.html#SYNOPSIS_section

Reply:
Refactored man page and CLI formatting to be mandoc-compliant.